lang/java/reef-tang/tang/src/test/java/org/apache/reef/tang/TestClassLoaders.java

```diff
@@ -20,36 +20,40 @@
 
 import org.apache.reef.tang.exceptions.BindException;
 import org.apache.reef.tang.exceptions.InjectionException;
-import org.apache.reef.tang.exceptions.NameResolutionException;
 import org.apache.reef.tang.types.ClassNode;
 import org.junit.Assert;
+import org.junit.Test;
 
 import java.io.File;
+import java.io.FilenameFilter;
 import java.net.MalformedURLException;
 
 public class TestClassLoaders {
 
-  //  @Test
-  public void testOneJar() throws MalformedURLException,
-      ClassNotFoundException, NameResolutionException, BindException {
+  private static final FilenameFilter JAR_FILE_FILTER = new FilenameFilter() {
+    public boolean accept(final File dir, final String name) {
+      return name.endsWith(".jar");
```

Comment:
I'm not sure this way of filtering is the best approach. It assumes that we can use any .jar file that happens to be present in that folder, while in fact we're looking for a very specific one. If, for example, needed jar file is missing, it's harder to debug what happened if we don't know which jar we were looking for.

In our .NET code, we're using constants for jar file names with version hardcoded, and update them during release using scripts. Maybe we could use the same approach here, just add this file to the list of files updated by release scripts?